target/scripts/helpers/utils.sh

```diff
@@ -84,27 +84,26 @@ function _replace_by_env_in_file
 {
   if [[ -z ${1+set} ]]
   then
-    _log 'warn' "(ldap.sh:_replace_in_file) missing first argument"
-    return 1
+    dms_panic__invalid_value 'first argument unset' 'utils.sh:_replace_by_env_in_file'
   elif [[ -z ${2+set} ]]
   then
-    _log 'warn' "(ldap.sh:_replace_in_file) missing second argument"
-    return 1
+    dms_panic__invalid_value 'second argument unset' 'utils.sh:_replace_by_env_in_file'
   elif [[ ! -f ${2} ]]
   then
-    _log 'warn' "(ldap.sh:_replace_in_file) file '${2}' could not be found"
+    _log 'warn' "(utils.sh:_replace_by_env_in_file) file '${2}' does not exist"
```

Comment:
Do we need a flag to toggle between logging a warning or panic failing for this? When would you want to prefer a logged warning instead of a hard fail when your usage references a file (or no file at all) that doesn't exist?

Reply:
Okay, okay, go convinced me; I will make it use panic-only and then we check for the file to exists before we call the method.